libpius/path.py

```diff
@@ -99,11 +99,21 @@ def set_tmpdir(dir):
     if sys.platform == "win32":
         TMP = os.environ.get('TEMP')
         return os.path.join(TMP, dir)
-    elif sys.platfrom == "darwin":
-        TMP = '/private/tmp/'
+    elif sys.platform == "darwin":
+        if os.environ.get('USER') == "root":
+            TMP = '/private/tmp/'
+        else:
+            TMP = os.environ.get('TMPDIR')
         return os.path.join(TMP, dir)
```

Comment:
Handling of macOS, during install root doesn't have a TMPDIR env variable so hard coded /private/tmp/ will take its place. When user runs program it should automatically get the TMPDIR location which is a secure directory that only that user has access to, not even root has access to it as only owner permissions are set. (only time root has access is if chown is used, even then I don't believe that is possible with the security profiles set in place by default)